src/epub/text/combat.xhtml

```diff
@@ -362,7 +362,7 @@
 			<p>But at least there were no restrictions on Paco and him.</p>
 			<p>They strolled up Gorky Street, jam packed with fellow pedestrians. Shoppers, window-shoppers, men on the prowl for girls, girls on the prowl for men, Ivan and his wife taking the baby for a stroll, street cleaners at the endless job of keeping Moscow’s streets the neatest in the world.</p>
 			<p>Paco pointed out this to Hank, Hank pointed out that to Paco. Somehow it seemed more than a visit to a western European nation. This was Moscow. This was the head of the Soviet snake.</p>
-			<p>And then Hank had to laugh inwardly at himself as two youngsters, running along playing tag in a grown-up world of long legs and stolid pace, all but tripped him up. Head of a snake it might be, but Moscow’s people looked astonishingly like those of Portland, Maine or Portland, Oregon.</p>
+			<p>And then Hank had to laugh inwardly at himself as two youngsters, running along playing tag in a grownup world of long legs and stolid pace, all but tripped him up. Head of a snake it might be, but Moscow’s people looked astonishingly like those of Portland, Maine or Portland, Oregon.</p>
```

Comment:
`grown-up` should be as an adjective. `grownup` is a noun.

Reply:
fixed